README.md

```diff
@@ -60,15 +64,19 @@ $validator = new EmailValidator();
 $validator->isValid("example@example.com", new RFCValidation()); //true
 ```
 
-
-### Available validations ###
+### Available validations
 
 1. [RFCValidation](/src/Validation/RFCValidation.php): Standard RFC-like email validation.
-2. [NoRFCWarningsValidation](/src/Validation/NoRFCWarningsValidation.php): RFC-like validation that will fail when warnings* are found.
-3. [DNSCheckValidation](/src/Validation/DNSCheckValidation.php): Will check if there are DNS records that signal that the server accepts emails. This does not entail that the email exists.
-5. [MultipleValidationWithAnd](/src/Validation/MultipleValidationWithAnd.php): It is a validation that operates over other validations performing a logical and (&&) over the result of each validation.
-6. [MessageIDValidation](/src/Validation/MessageIDValidation.php): Follows [RFC2822 for message-id](https://tools.ietf.org/html/rfc2822#section-3.6.4) to validate that field, that has some differences in the domain part.
-7. [Your own validation](#how-to-extend): You can extend the library behaviour by implementing your own validations.
+2. [NoRFCWarningsValidation](/src/Validation/NoRFCWarningsValidation.php): RFC-like validation that will fail when
+   warnings* are found.
```

Comment:
I don't see this improving the reading. Please keep the original format. Apply this comment to all the lines below.

Reply:
Refer to https://github.com/egulias/EmailValidator/pull/364#discussion_r1212802161.